apps/src/lab2/views/components/PanelContainer.tsx

```diff
@@ -1,16 +1,24 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 import moduleStyles from './panelContainer.module.scss';
 
+interface PanelContainerProps {
+  id: string;
+  headerText: string;
+  children?: React.ReactNode;
+  rightHeaderContent?: React.ReactNode;
+  leftHeaderContent?: React.ReactNode;
+  isHeadersShowing?: boolean;
```

Comment:
Discussed offline, but it may be clearer to invert this to `hideHeaders`, so that the default state of showing header is more clear. Can be a follow up

Reply:
I'll add this as a follow-up since it may mean updating music lab files. Thanks!

Reply:
On second thought - I'll go ahead and update now. Thanks!